txclib/project.py

```diff
@@ -108,32 +108,43 @@ def getset_host_credentials(self, host, username=None, password=None,
 
         save = False
         config_username, config_password = None, None
+        if 'TX_TOKEN' in os.environ:
+            # User api info from environment variable
+            password = os.environ['TX_TOKEN']
+            username = 'api'
+            
+            """We need to check if hostname info exists in the .transifexrc file
+            If not, it means that this is the first time this function runs, so
+            we need to create its data.
+            """
+            try:
+                self.txrc.get(host, 'hostname')
+            except(configparser.NoOptionError, configparser.NoSectionError):
```

Comment:
@SofiaMargariti calling the _update_file_host() will notify the user for "Updating the .transifextx file" on each pull / push (since the code will run every time, and not just once).

I think for the same reason  the similar try/except exists on line 134 (for username/password).

I can move the warning outside the _update_file_host() function, but this means that there will be duplicate code. Whatever you think is best :)